plugins/modules/junos_package.py

```diff
@@ -52,6 +52,12 @@
       to the remote device prior to installing.
     type: bool
     default: no
+  unlink:
+    description:
+    - The I(unlink) argument is responsible for instructing the remote device to
+    remove the package after installation.
```

Comment:
The indentation issue here is resulting in a traceback from collection_prep's doc generation tool. As a result, a lot of the RSTs are getting deleted. Could you please fix this and push the changes? Thank you.